src/Azure.Functions.Cli/Actions/LocalActions/CreateFunctionAction.cs

```diff
@@ -376,7 +376,7 @@ private void ConfigureAuthorizationLevel(Template template)
             else
             {
                 var binding = bindings.Where(b => b["type"].ToString().Equals(HttpTriggerTemplateName, StringComparison.OrdinalIgnoreCase)).FirstOrDefault();
-                binding["authLevel"] = AuthorizationLevel.ToString();
+                binding["authLevel"] = AuthorizationLevel.ToString().ToLowerInvariant();
```

Comment:
@mattchenderson Is lower case good enough? I see you mentioned "camelCase" in the issue.

Reply:
Lowercasing will work for the existing values, but the correct fix for this would be to camel case.

That said, we can merge this as-is, as the likelihood of us adding something else at this point is low, and if we did, we'd get another warning.